apiclient.js

```diff
@@ -1,4 +1,4 @@
-﻿define(['events'], function (events) {
+﻿define(['events', "jsencrypt", 'text!../Secure', "cryptojs-aes"], function (Events, Crypto, PublicKey) {
```

Comment:
This should be changed to require when needed, not when loading the script file. Most of the time we'll be using saved authentication so we can avoid the cost of loading these resources unless a login takes place.